sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -344,6 +344,9 @@ class ShareProperties(DictMixin):
     :ivar bool enable_snapshot_virtual_directory_access:
         Specifies whether the snapshot virtual directory should be accessible at the root of the share
         mount point when NFS is enabled. if not specified, the default is True.
+    :ivar bool paid_bursting_enabled: This property enables paid bursting.
+    :ivar int paid_bursting_max_bandwidth_mips: The maximum throughput the file share can support.
```

Comment:
Typo here `mips` -> `mibps`. This one can be fixed without the fix to generated code.

Additionally can we add "in Mbps" to the end of the description here and everywhere we have the description.

Reply:
I think you missed this one specific line for the typo and update to docs MiB/s. Also, I think it should be `Mib/s` (mebibits per second). Capital `B` means mebibytes.

Reply:
Nice catch!